Documentation for the extent param in Package Layers algorithm (#10012)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/database.rst

       setting this option to ``True`` will result in exporting also its related layer(s).
       If the layer has features selected, then only their related features will be exported
       unless the related layer was also an input layer.
   * - **Extent**

       Optional
     - ``EXTENT``
     - [extent]

>      - Limit exported features to those with geometries intersecting the
>        provided extent. If none of the features of a specific input layer intersect the
>        provided extent, the layer will still be created and will be empty in the GeoPackage
>        file.

Reviewer: ```suggestion
     - Limit the exported features to those with geometries that intersect the provided 
       extent. If none of the features from a specific input layer intersect the extent, 
       the layer will still be created in the GeoPackage file, but it will be empty.
```
